pymc_marketing/clv/models/gamma_gamma.py

```diff
@@ -174,7 +174,7 @@ class GammaGammaModel(BaseGammaGammaModel):
     data: pd.DataFrame
         DataFrame containing the following columns:
             - customer_id: Customer labels. Must not repeat.
-            - mean_transaction_value: Mean transaction value of each customer.
+            - monetary_value: Average transaction value for each customer.
```

Comment:
I changed it to `monetary_value` so that results from the `rfm_summary` utility can be used without renaming anything. However, `GammaGammaModelIndividual` requires raw transaction values instead, so I think it's important to keep the distinction.

Reply:
> However, GammaGammaModelIndividual requires raw transaction values instead, so I think it's important to keep the distinction.

Why call the individual transactions values, but the aggregated monetary value? It's the same quantity, but one of them is the average of the other. I don't see why it should have a different label

Reply:
Monetary Value is the term used in research and every educational article/webinar about CLV, so I'm loathe to change it. `GammaGammaIndividual` is also rather niche; majority of users will be doing population modeling.

Reply:
Fine but then change the Gamma individual name to sound more like monetary value